Source/WebCore/page/Settings.yaml

```diff
@@ -115,8 +115,8 @@ FixedBackgroundsPaintRelativeToDocument:
   type: bool
   defaultValue:
     WebCore:
-      PLATFORM(IOS_FAMILY): true
-      default: false
+      PLATFORM(IOS_FAMILY): false
```

Comment:
Why did you flip this around? "fixedBackgroundsPaintRelativeToDocument" is the iOS behavior

Reply:
Switched to "cssFixedBackgroundSupport" for clarity.